docs/articles/opinion/importance_of_governance.md

```diff
@@ -0,0 +1,60 @@
+# The importance of Governance
+> **Why it matters and why you should care**
+
+_Author [Diego Xirinachs / @dxiri](https://github.com/dxiri)_
+
+![](images/dao.jpg)
+
+Unstoppable Organizations!
+
+That is Aragon's premise, but how on earth can you build such a thing?
+
+You need a set of rules that everyone **must** follow in order to achieve any given organization goals. The must is the hard part, people do things for different reasons, and interests not always align,
+so you need a system that can incentivize good behavior while punishing bad behavior.
+
+Sounds familiar? This is what governments all around the world do. If you behave like a good citizen (by paying your
+taxes, for example) you are "rewarded" by allowing you to get a job, get a government issued ID/drivers license, etc.
+
+If you don't behave, you go to to jail, get fined and/or barred from certain communities, for example if you are a
+doctor and kill someone because of negligence, you get expelled from the medic community and forbidden to exercise
+your profession.
+
+Enter governance.
+
+To start, it may be good to define what governance is and then we can drill down into why it is important.
+
+From Wikipedia "Governance is all of the processes of governing, whether undertaken by a government,
+a market or a network, over a social system (family, tribe, formal or informal organization, a territory or across territories)
+and whether through the laws, norms, power or language of an organized society.
+
+It relates to "the processes of interaction and decision-making among the actors involved in a collective problem that lead to
+the creation, reinforcement, or reproduction of social norms and institutions."
+
+In lay terms, it could be described as the political processes that exist in between formal institutions."
+
+That last line, pretty much sums it up.
+
+If you don't have any governance, you have an anarchy, where there is no direction or common goals, causing confusion and disorder.
```

Comment:
The word "anarchy" mean "no rulers", not "no rules". On the contrary, anarchists have developed highly evolved governance systems.

See:

https://theanarchistlibrary.org/library/the-anarchist-faq-editorial-collective-an-anarchist-faq-02-17#toc2

https://c4ss.org/wp-content/uploads/2014/01/Anarchist-Themes-in-the-Work-of-Elinor-Ostrom.pdf